3drepo/3drepo.io
# ISSUE #3344 - Realtime events - Update container / new revision

This fixes #3344

#### Description
- Adds realtime event for when the void status of a revision is changed
- Adds a realtime event for when a new revision is uploaded
    - This updates the revision count and last updated value on the container list item
    - This Adds the revision to the revision list for highlighted containers

#### Test cases
-   Open 3d repo in two accounts at the same time
-   Navigate to the containers list
-   In one instance of 3drepo change the void status of a revision and observe it change in the other instance
-  In one instance of 3D repo upload a new revision. When the revision is done processing (Note: this is different to the initial upload in the upload revision modal) observe it appearing in the other instance
